src/components/scenes/newAccount/NewAccountTosScene.tsx

```diff
@@ -77,13 +78,60 @@ export const NewAccountTosScene = (props: Props) => {
 
   const handleNextPress = () => {
     logEvent(`Signup_Terms_Agree_and_Create_User`)
-    dispatch(
-      createUser({
-        username,
-        password,
-        pin
-      })
-    )
+    const { username, password, pin } = route.params
+    if (pin == null) throw new Error('No PIN set')
+    const { context } = imports
+    dispatch({
+      type: 'NAVIGATE',
+      data: {
+        name: 'newAccountWait',
+        params: {
+          title: s.strings.great_job,
+          message: s.strings.hang_tight + '\n' + s.strings.secure_account
+        }
+      }
+    })
+    setTimeout(async () => {
+      try {
+        const abcAccount = await context.createAccount({
```

Comment:
Please rename `abcAccount` to just `account`. The "ABC" prefix stands for "AirBitz Core", and is super-duper out-dated.